raspberry/bzzz/estimators/altitude_Kalman_filter.py

```diff
@@ -22,26 +36,26 @@ def __init__(self, sampling_frequency=10, initial_Tt=0., x_tilde_0=np.zeros((4,
         self.__C = np.array([[1, 0, 0, 0]])
 
         # Process noise
-        self.__Q = np.eye(4, 4)*1e-5
+        self.__Q = np.diagflat([1e-4, 1, 5, 1e-3])
         # Measurement noise
-        self.__R = 1e-3
+        self.__R = 0.1
 
         # initial conditions
         self.__x_hat_0_minus1 = x_tilde_0
         self.__sigma_0_minus1 = P_0
-        
+
         # Measurement update
-        self.__x_MU = None
-        self.__sigma_MU = None
+        self.__x_MU = overwrite_x_MU
+        self.__sigma_MU = overwrite_sigma_MU
 
         # Time update
         self.__x_TU = x_tilde_0
         self.__sigma_TU = P_0
 
-        # Point to a function that does nothing if caching is not enabled.
-        self.__cache_MU_values = lambda : ()
-        self.__cache_TU_values = lambda : ()
+        self.__cache_MU_values = lambda: ()
```

Comment:
I still find this overly complicated. These are caches and caches are arrays (or lists if you prefer) - not lambdas.

Reply:
These are functions, which when called cache values into lists. If we configure to cache values, they will point to a function that does cache, otherwise, they point to this empty function. I will change these to be less complicated.

Reply:
Done!